app/presenters/job_presenter.rb

```diff
@@ -2,7 +2,7 @@ module JobPresenter
   class << self
     def single(job)
       payload = job['payload']
-      return if job['payload'].blank? || payload['created_from'] != 'API'
+      return if (job['payload'].blank? || payload['created_from'] != 'API') && (job['payload'].blank? || payload['created_from'] != 'BackendSpecialCase')
```

Comment:
Somehow the term `BackendSpecialCase` doesnt look too good to me.
Can we rename it something like `backend_flow` which clealy says that its a flow created by another backend flow.

@shirshendu ???